src/backend/backend-agnostic.md

```diff
@@ -2,11 +2,12 @@
 
 <!-- toc -->
 
-As of <!-- date-check --> Aug 2022, `rustc_codegen_ssa` provides an
-abstract interface for all backends to implement, to allow other codegen
-backends (e.g. [Cranelift]).
+[`rustc_codegen_ssa`]
+provides an abstract interface for all backends to implement,
+to allow other codegen backends (e.g. [Cranelift]).
```

Comment:
While we're at it, what do you think about changing the wording to this?

```suggestion
[`rustc_codegen_ssa`]
provides an abstract interface for all backends to implement,
enabling the creation of new backends (e.g. [Cranelift]).
```

Reply:
how about "to allow code sharing (e.g. with [Cranelift])"

Reply:
Actually that doesn't sound right. I think we should just remove the part starting with "enabling", or just mention all 3 backends?

Reply:
see if this is resolved with https://github.com/rust-lang/rustc-dev-guide/pull/1606/commits/dfb468a52fb606b6647e7240756f2c399654f179